.github/workflows/changelog.yml

```diff
@@ -55,9 +55,13 @@ jobs:
           git add *
           git commit -m "misc: update Changelog" -a
 
-      - name: Push changes
-        if: ${{ steps.pr2changelog.outputs.generated_changelog == 1}}
-        uses: ad-m/github-push-action@master
+      - name: Pushes to another repository
+        uses: cpina/github-action-push-to-another-repository@main
+        env:
+          API_TOKEN_GITHUB: ${{ secrets.API_TOKEN_GITHUB }}
         with:
-          force: true
-          github_token: ${{ secrets.PR2CHANGELOG_TOKEN }}
+          source-directory: 'output'
+          destination-github-username: 'US13-BOT'
```

Comment:
use the same user we were using, also is this github username or git username? Because those are different and we need an account if the former.